bindings/ros/aditof_roscpp/include/aditof_roscpp/generic_msg.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GENERIC_MSG_H
+#define GENERIC_MSG_H
+
+#include <aditof/camera.h>
+
+class GenericMsg {
```

Comment:
Since classes will inherit from this one, this class must have a virtual destructor!